components/inputs/datetimeField/TimeInput.tsx

```diff
@@ -130,27 +132,44 @@ const TimeInput = forwardRef<HTMLInputElement, TimeInputHtmlProps>(
         return;
       }
 
-      const validAsTime = isStringValidAsTime(reformatedNewValue);
-      setError(!validAsTime);
-
-      if (validAsTime) {
-        setError(!isTimeWithinEdges(reformatedNewValue, min, max));
+      if (isStringValidAsTime(reformatedNewValue)) {
         setInputValue(reformatedNewValue);
       }
     };
 
     const handleBlur = () => {
-      console.log('toto');
       if (!inputValue) {
         onChange(null);
         setError(false);
+        if (setTimeError) setTimeError(null);
```

Comment:
```suggestion
        if (setTimeError) {
            setTimeError(null);
        }
```
petite convention de code de l'équipe ;-)